.github/workflows/build.yml

```diff
@@ -45,6 +45,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Docker Hub login
+        if: ${{ !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork) }}
+        run: echo "${{ secrets.DOCKERHUB_TOKEN }}" | docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" --password-stdin
```

Comment:
Why did you add this commit?

I guess it was a mistake but it could look like malicious code to reveal secrets.

Reply:
Yeah it was a mistake, stack overflow has hacky suggestions sometimes.